tempdb_test.go

```diff
@@ -1,11 +1,46 @@
 package tempdb
 
 import (
+	"log/slog"
+	"os"
+	"strconv"
 	"testing"
 
 	"github.com/btcsuite/btcwallet/walletdb/walletdbtest"
 )
 
 func TestInterface(t *testing.T) {
+	Logger = slog.Default()
+
 	walletdbtest.TestInterface(t, "tempdb", "test.db")
 }
+
+func TestForEach(t *testing.T) {
+	h := slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{Level: slog.LevelDebug})
```

Comment:
this should be done in `TestMain` function, so it's shared for every test.  this should also log to `os.Stderr` (used for errors/warnings) not `os.Stdout` (used for outputs).

Reply:
done in https://github.com/linden/tempdb/pull/1/commits/d22c8bb7be7b2659be6c24617b1f5f2c9166856f.